src/main/java/net/minestom/server/listener/BlockPlacementListener.java

```diff
@@ -128,6 +129,8 @@ public static void listener(ClientPlayerBlockPlacementPacket packet, Player play
             return;
         }
 
+        if(placementPosition.y() >= BLOCK_HEIGHT) return;
```

Comment:
I meant moving it to line 92, in which case the events would not be called, but in my opinion they shouldn't be in this case